megamek/src/megamek/common/Entity.java

```diff
@@ -9350,10 +9356,19 @@ public String statusToString(int loc) {
             return "No location given.";
         }
 
-        return getLocationName(loc) + " (" + getLocationAbbr(loc)
+        String str = getLocationName(loc) + " (" + getLocationAbbr(loc)
```

Comment:
I realized you inherited some of this logic, but one thing to consider going forward (not asking for a fix here) is that one-shot ammo is mounted at LOC_NONE and won't be in the crits array. Perhaps a separate command or something could get you an ammo inventory for the entity.

Reply:
Honestly I am not happy with how it gets reported out and I am tryign to find a more usefull output, but this is a stopgap for something to be noted to be a semi-blocker to play. So this should be flagged as something to improve urgently but this is at least working.